api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/products_controller.rb

```diff
@@ -25,7 +25,11 @@ def index
 
   # GET /mnoe/jpi/v1/admin/products/id
   def show
-    @product = MnoEnterprise::Product.find_one(params[:id], DEPENDENCIES)
+    @product = MnoEnterprise::Product
```

Comment:
Do we need to deal with the scenario where `editAction` is not specified?

Reply:
@BrunoChauvet So there is guarding on the adapter bluesky for when the edit_action isn't specified -- nothing should break on mnoe when its not specified, but we may want to put a default here as well.